README.md

```diff
@@ -66,28 +70,30 @@ The `/store` volume of the container is used for persistent storage of location
 data. The volume needs to be created explicitly.
 
 ```bash
-$ docker volume create recorder_storage
-$ docker run -d -p 8083:8083 -v recorder_store:/store owntracks/recorder
+docker volume create recorder_storage
+docker run -d -p 8083:8083 -v recorder_store:/store owntracks/recorder
 ```
+
 It is also possible to use a local folder instead of an static docker volume.
 
 ```bash
-$ mkdir store
-$ docker run -d -p 8083:8083 -v ./store:/store owntracks/recorder
+mkdir store
+docker run -d -p 8083:8083 -v ./store:/store owntracks/recorder
 ```
 
 If nothing is mounted at `/store`, docker will create a unique volume
 automatically. However up on recreation of the docker container, this process
 will be repeated and another unique volume will be created. As a result, the
 container will have forgotten about previous tracks.
 
-## TLS between MQTT broker and Recorder
+## TLS between MQTT Broker and Recorder
```

Comment:
I'm not typically a great fan of uppercased nouns in English, also not in titles, but it'll pass. :)

Reply:
:D I dont have a preference here. I just noticed the inconsistency between **b**roker and **R**ecorder which triggered my OCD.

Reply:
Ah, yes, that's actually on purpose: I say Recorder to mean `ot-recorder` as in a bit of a proper name.
But fine; leave it as you've done.